content/updates/2020-08/3-API-option-labels.md

```diff
@@ -0,0 +1,22 @@
+---
+pim_announcement_img: ./img/option-labels.png
+pim_announcement_alt_img: "Option labels inside product endpoint"
+pim_announcement_audience:
+- CE
+- EE
+---
+
+# Get options labels directly inside the products endpoint
+::: meta-data type="New" features="API, Connectivity" available="late August" link-to-doc="https://api.akeneo.com/api-reference.html#get_products"
+:::
+
+We improved our API to help you reducing the number of API calls needed to synchronize product data.
```

Comment:
```suggestion
We improved our API to help you reduce the number of API calls needed to synchronize product data.  
```
Please, do not forget those two spaces at the end of the line.

Reply:
Can we mention somewhere why we added specifically those labels? Because they usually need them to be displayed in their e-commerce front I guess?

Reply:
Or the PDF example you gave in Slack could work too.

Reply:
I launched the help center on my computer and it's a bit weird with a line break.
I also added the PDF example that I gave you in Slack. 
But I don't really know how to format the attribute list with codes and labels. It will be super long if I use a line for each code then a new line per label. What do you think?